### PR TITLE
chore: add window templates

### PR DESCRIPTION
**Pull request by zhpenkov**
Adds common templates for the Window component & update tests.

### CODE REVIEW

**Comment by inikolova** [comment]
In my opinion we don't need **window-with-action-buttons** template; we should demonstrate this option in a test (similar to the [Dialog component](https://github.com/telerik/kendo-themes/blob/develop/packages/html/src/dialog/tests/dialog.tsx)).

File: tests/_output/bootstrap/window/window-rtl.png
Comment (inikolova) [inline tests/_output/bootstrap/window/window-rtl.png:1]: Title is cropped - please check

File: packages/html/src/window/tests/window.tsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
-import { Window } from '../../window';
+import { WindowNormal } from '..';
 
 const style = `
     .k-window {
         position: relative;
+        width: 300px;
```
Comment (inikolova) [inline packages/html/src/window/tests/window.tsx:6]: Maybe we could increase this width?

Comment (zhpenkov) [reply packages/html/src/window/tests/window.tsx]: I decided to remove the width limitation completely as it was and leave it to scale according to the content it has. **WDYT**.

File: packages/html/src/editor/tests/editor-table-wizard.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ export default () =>(
         <style>{styles}</style>
         <div id="test-area" className="k-d-grid k-grid-cols-2">
             <section>
-                <Window className="k-editor-window k-editor-table-wizard-window" title="Table properties" actions={[ 'minimize', 'maximize', 'close' ]} actionButtonsAlign="start" actionButtons={
```
Comment (inikolova) [inline packages/html/src/editor/tests/editor-table-wizard.tsx:25]: Editor table wizard window buttons differ from the WindowNormal default buttons

**Review by inikolova — APPROVED** [review]

**Pull request merged**